tests/libnode/node.go

```diff
@@ -351,3 +362,35 @@ func SetNodeUnschedulable(nodeName string, virtCli kubecli.KubevirtClient) {
 func SetNodeSchedulable(nodeName string, virtCli kubecli.KubevirtClient) {
 	setNodeSchedualability(nodeName, virtCli, true)
 }
+
+func GetVirtHandlerClient(virtCli kubecli.KubevirtClient) kubecli.VirtHandlerClient {
+	namespace, err := kubevirtutil.GetNamespace()
+	util.PanicOnError(err)
+
+	clientConfig, err := kubecli.GetKubevirtClientConfig()
+	util.PanicOnError(err)
+
+	aggregatorClient := aggregatorclient.NewForConfigOrDie(clientConfig)
+
+	kubeInformerFactory := controller.NewKubeInformerFactory(virtCli.RestClient(), virtCli, aggregatorClient, namespace)
```

Comment:
I thought about this again, I don't think we can or should do this.

This basically starts an informer for the whole remaining runtime of the test binary.

I just double-checked how we use this client in the tests: We just use it to get the virt-handler pod.

It should be sufficient for the tests to create the handler client like this:

```
kubecli.NewVirtHandlerClient(virtCli, &http.Client{})
```

That should basically be the whole content of the function.

Reply:
> We just use it to get the virt-handler pod.

Since it is the case, perhaps it would make sense to have a function `GetVirtHandlerPod` instead of `GetVirtHandlerClient`. That will prevent from using the created client in a wrong way.

Reply:
Thanks!
Done with with of your suggestions. Now the function is called `GetVirtHandlerPod()`.